rusk-wallet/src/bin/command.rs

```diff
@@ -944,6 +996,9 @@ impl fmt::Display for RunResult {
                 let reward = Dusk::from(data.reward);
                 write!(f, "> Accumulated reward is: {reward} DUSK")
             }
+            ContractId(bytes) => {
+                write!(f, "> Contract ID: {:?}", bytes)
```

Comment:
this needs to be displayed as 64-character hexadecimal (currently it is an array if decimal bytes)

Reply:
@miloszm Using hex::encode() now